### DISCUSSION TITLE
Icon Color for Support Buttons

### DISCUSSION THREAD

**Discussion by johnpernock** [Clooos/Bubble-Card#574] [Feature requests]
I have a card setup for my printer, and I have 4 sub buttons to show the ink status. I would love to be able to color the icons for each to match the color but I still haven't found a way to do that and wanted to know if thats possible?
![Screenshot 2024-06-11 at 10 13 15 AM](https://github.com/Clooos/Bubble-Card/assets/2034917/6d4b45ba-21ed-466d-9087-047c45159203)

**Comment by johnpernock** [comment]
I was able to figure this out with some css instead.

**Reply by Clooos** [reply]
This is the way indeed! 👌
